src/components/vote-form.js

```diff
@@ -13,45 +13,42 @@ function VoteForm() {
 
 	const getCandidates = async () => {
 		await axios
-			.get(process.env.API_HOST + '/candidates/', candi)
+			.get(process.env.API_HOST + '/candidates/', candidates)
 			.then(({ data }) => {
-				setCandi(data);
+				setCandidates(data);
 			})
-			.catch(function (error) {
+			.catch((error) => {
 				console.log(error);
 			});
 	};
 
-	const voteCandidates = async (person) => {
-		const newUrl =
-			process.env.API_HOST + '/candidates/' + person._id + '/vote/';
+	const voteCandidates = async (candidate) => {
+		const newUrl = `${process.env.API_HOST}/candidates/${candidate._id}/vote/`;
 		await axios
 			.put(newUrl, voteCount)
 			.then(({ data }) => {
 				setVoteCount(data);
-				alert(person.name + '님에게 투표 완료!');
+				alert(candidate.name + '님에게 투표 완료!');
 				getCandidates();
 			})
 			.catch(function (error) {
 				console.log(error);
 				alert('투표 실패!');
 			});
 	};
-	let i = 1;
+
 	return (
 		<Wrapper>
 			<Title>
 				<RedTitle>프론트엔드 인기쟁이</RedTitle>는 누구?
 			</Title>
 			<SubTitle>CEOS 프론트엔드 개발자 인기 순위 및 투표 창입니다.</SubTitle>
 			<VoteArea>
-				{candi
-					.sort((person1, person2) => {
-						return person2.voteCount - person1.voteCount;
-					})
-					.map((person) => (
-						<Row key={JSON.stringify(person)}>
-							<Rank>{i++}위:</Rank>
+				{candidates
+					.sort((person1, person2) => person2.voteCount - person1.voteCount)
+					.map((person, index) => (
+						<Row key={person._id}>
+							<Rank>{index + 1}위:</Rank>
 							<CandiName>
 								{person.name}
 								<br />[{person.voteCount}표]
```

Comment:
candidate-card.js 이름으로 새 파일을 만들어서, CandidateCard라는 새로운 컴포넌트 작성한 뒤에
이 부분에서 아래와 같이 쓸 수 있도록 리팩토링 해보는건 어떨까요?
```suggestion
					.map((candidate, index) => (
						<CandidateCard {...candidate} />
					))
```